src/Uno.UI/DataBinding/BindingPropertyHelper.cs

```diff
@@ -753,9 +753,6 @@ private static ValueSetterHandler InternalGetValueSetter(Type type, string prope
 				}
 			}
 
-			var propertyInfo = Uno.Funcs.CreateMemoized(() => GetPropertyInfo(type, property, allowPrivateMembers: false));
```

Comment:
Why do you need to remove this ?

Reply:
It's just a minor optimization I did : I moved it in the inner scope to reduce useless allocation, and I also removed the memoized deferring as it was used only once directly in those scope, again to reduce useless allocation.

Reply:
The first one is probably some leftover optimization, you moved the second one which is fine